### PR TITLE
testing/liberasurecode: new aport

### PR DESCRIPTION
**Pull request by ehudkaldor**
https://github.com/openstack/liberasurecode
Erasure Code API library written in C with pluggable Erasure Code backends
testing/

### CODE REVIEW

**Comment by ehudkaldor** [comment]
Hi,
as this is my first time submitting a PR to Aports - how long does it usually take for PR to get reviewed?

File: testing/liberasurecode/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Contributor: Ehud Kaldor <ehud@unfairfunction.org>
+# Maintainer: Ehud Kaldor <ehud@unfairfunction.org>
+pkgname=liberasurecode
+pkgver=1.5.0
+pkgrel=0
+pkgdesc="Erasure Code API library written in C with pluggable Erasure Code backends"
+url="https://github.com/openstack/liberasurecode"
+arch="all"
+license="COPYING"
```
Comment (jirutka) [inline testing/liberasurecode/APKBUILD]: I don’t know any license named COPYING and it’s definitely not any SPDX license...

**Comment by ehudkaldor** [comment]
The license file is a file names COPYING inside the package. When I tried
to put the full text the build broke in me. What's the right way to do that?

**Comment by george-hopkins** [comment]
@ehudkaldor  The `license` field usually contains a [SPDX identifier](https://spdx.org/licenses/). Judging from `COPYING`, `BSD-2-Clause` would be appropriate in this case.

**Comment by ehudkaldor** [comment]
@jirutka done

File: testing/liberasurecode/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Maintainer: Ehud Kaldor <ehud@unfairfunction.org>
+pkgname=liberasurecode
+pkgver=1.5.0
+pkgrel=0
+pkgdesc="Erasure Code API library written in C with pluggable Erasure Code backends"
+url="https://github.com/openstack/liberasurecode"
+arch="all"
+license="COPYING" 
+depends=""
+makedepends="libtool make automake autoconf gcc musl-dev zlib-dev"
```
Comment (ncopa) [inline testing/liberasurecode/APKBUILD]: you dont need `make`, `gcc` or `musl-dev` in make depends as those are pulled in by the implicit `build-base`. Please remove those.

File: testing/liberasurecode/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="${pkgname}-${pkgver}-src.tar.gz::https://github.com/openstack/liberasurecode/archive/1.5.0.tar.gz"
```
Comment (ncopa) [inline testing/liberasurecode/APKBUILD]: Please use `$pkgver` which makes it easier to maintain it (upgrade means simply change `pkgver`)
````sh
source="${pkgname}-${pkgver}-src.tar.gz::https://github.com/openstack/liberasurecode/archive/$pkgver.tar.gz"
````

Comment (ehudkaldor) [reply testing/liberasurecode/APKBUILD]: done, I think...

**Comment by ehudkaldor** [comment]
@ncope done

File: testing/liberasurecode/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var
+	make
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" install
+	install -Dm644 COPYING "$pkgdir"/usr/share/licenses/$pkgname/COPYING
```
Comment (jirutka) [inline testing/liberasurecode/APKBUILD]: This is just text of the BSD-2-Clause license without any additional information, so including it in the package is unnecessary.

**Comment by ehudkaldor** [comment]
any update on this?

**Comment by ehudkaldor** [comment]
hi @ncopa ,
how do I mark your comments as done? they still show up as requested changes

**Comment by ehudkaldor** [comment]
bump?

**Comment by clandmeter** [comment]
Please rebase this against master to trigger CI

**Comment by ehudkaldor** [comment]
done

**Comment by ehudkaldor** [comment]
never worked with Drone before, so can anyone tell me why it's failing?

**Comment by Ikke** [comment]
The commits in this PR are prettty messy and conflict when applied (same when I try locally). So it would help to clean up the commits a bit and remove the unnecessary merge commits from the PR.

**Pull request closed without merging**